pytest_rpc/helpers.py

```
@@ -216,10 +217,162 @@ def delete_instance(instance_name, run_on_host):
+        instance_name (str): The name of the OpenStack instance from which the snapshot is created.
+        run_on_host (testinfra.Host): Testinfra host object to execute the action on.
+    """
+    instance_id = get_id_by_name('server', instance_name, run_on_host)
+    cmd = "{} openstack server image create --name {} {}'".format(utility_container, snapshot_name, instance_id)
+
+    run_on_host.run_expect([0], cmd)
+    sleep(10)
+
+
+def delete_it(service_type, service_name, run_on_host):
+    """Delete an OpenStack object
+
+    Args:
+        service_type (str): The OpenStack object type to query for.
+        service_name (str): The name of the OpenStack object to query for.
+        run_on_host (testinfra.Host): Testinfra host object to execute the action on.
+
+    Raises:
+        AssertionError: If operation is unsuccessful. """
+
+    sleep(15)
+    id = get_id_by_name(service_type, service_name, run_on_host)
+    cmd = "{} openstack {} delete {}'".format(utility_container, service_type, id)
+    run_on_host.run_expect([0], cmd)
+    sleep(20)
+
+    assert not verify_asset_in_list(service_type, service_name, run_on_host)
+
+
+def create_floating_ip(network_name, run_on_host):
+    """Create floating IP on a network
+
+    Args:
+        network_name (str): The name of the OpenStack network object on which the floating IP is created.
+        run_on_host (testinfra.Host): Testinfra host object to execute the action on.
+
+    Raises:
+        ValueError: If it has the right type but an inappropriate value.
+
+    Returns:
+        str: The newly created floating ip name
+    """
+
+    network_id = get_id_by_name('network', network_name, run_on_host)
+    assert network_id is not None
+
+    cmd = "{} openstack floating ip create {} -f json'".format(utility_container, network_id)
+    output = run_on_host.run(cmd)
+
+    assert (output.rc == 0)
+
+    try:
+        result = json.loads(output.stdout)
+    except ValueError:
```

Do you want to swallow this error?  Your doc string says that this function will raise this error.

---

updated docstring in the next commit